src/embedded/embeddedManager.ts

```
@@ -27,8 +28,10 @@ export class IterableEmbeddedManager {
     callback: () => void,
     placementIds?: number[]
   ) {
-    await this.retrieveEmbeddedMessages(packageName, placementIds || []);
-    callback();
+    if (typeOfAuth !== null) {
+      await this.retrieveEmbeddedMessages(packageName, placementIds || []);
+      callback();
+    }
```

should we throw an error in the case of no auth?